Sources/Lumina/Camera/Extensions/SampleBufferExtension.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SampleBufferExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+extension CMSampleBuffer {
+  func normalizedStillImage(forCameraPosition position: CameraPosition) -> UIImage? {
+    LuminaLogger.notice(message: "normalizing still image from CMSampleBuffer")
+    guard let imageData = AVCapturePhotoOutput.jpegPhotoDataRepresentation(forJPEGSampleBuffer: self, previewPhotoSampleBuffer: nil) else {
+      return nil
+    }
+    guard let dataProvider = CGDataProvider(data: imageData as CFData) else {
+      return nil
+    }
+    guard let cgImageRef = CGImage(jpegDataProviderSource: dataProvider, decode: nil, shouldInterpolate: true, intent: CGColorRenderingIntent.defaultIntent) else {
+      return nil
+    }
+    return UIImage(cgImage: cgImageRef, scale: 1.0, orientation: getImageOrientation(forCamera: position))
+  }
+  
+  func normalizedVideoFrame() -> UIImage? {
+    LuminaLogger.notice(message: "normalizing video frame from CMSampleBbuffer")
+    guard let imageBuffer = CMSampleBufferGetImageBuffer(self) else {
+      return nil
+    }
+    let coreImage: CIImage = CIImage(cvPixelBuffer: imageBuffer)
+    let context: CIContext = CIContext()
+    guard let sample: CGImage = context.createCGImage(coreImage, from: coreImage.extent) else {
+      return nil
+    }
+    return UIImage(cgImage: sample)
+  }
+  
+  private func getImageOrientation(forCamera: CameraPosition) -> UIImage.Orientation {
+    switch UIApplication.shared.statusBarOrientation {
+      case .landscapeLeft:
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/SampleBufferExtension.swift#L42"></span>Case statements should vertically align with their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>SampleBufferExtension.swift:42</code></td>
    </tr>
  </tbody>
</table>